### DISCUSSION TITLE
如何在valaxy中使用twikoo提供的评论统计，并将其显示在文章上？

### DISCUSSION THREAD

**Discussion by MengNianxiaoyao** [YunYouJun/valaxy#403] [Q&A]
如题，想要在这里
![image](https://github.com/YunYouJun/valaxy/assets/103258286/98ab8241-d56b-4da6-8ad0-2607c7d36b5c)
以及这里
![Clip_2024-07-03_09-24-06](https://github.com/YunYouJun/valaxy/assets/103258286/8c10f5ed-6c24-4bfd-bd65-5f13611520df)
添加

**Comment by YunYouJun** [comment]
如果需要在首页展示评论统计，意味着需要查询首页的所有的文章的对应评论统计，再更新至首页卡片对应位置。

目前想到的一种是，进入列表页批量查询更新。（查询次数较多）
另一种是在生成文章时将数据插入到 meta.frontmatter（静态化，无法实时更新。）

---

但我自己并没有使用 twikoo，所以这个功能理论上应该并没有做（可以 PR

**Comment by MengNianxiaoyao** [comment]
列表页可以不做

文章页可以加载文章的时候进行实时查询，这部分应该容易做，但我实操的时候老是出问题
